src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int thirdLeg;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int thirdLeg) {
```

Comment:
Update the constructor parameter names accordingly, after renaming 'thirdLeg' to 'height'.